cmd/outline-ss-server/main.go

```diff
@@ -341,12 +310,13 @@ func (s *SSServer) Stop() error {
 }
 
 // RunSSServer starts a shadowsocks server running, and returns the server or an error.
-func RunSSServer(filename string, natTimeout time.Duration, sm *outlineMetrics, replayHistory int) (*SSServer, error) {
+func RunSSServer(filename string, natTimeout time.Duration, serverMetrics *serverMetrics, serviceMetrics service.ServiceMetrics, replayHistory int) (*SSServer, error) {
```

Comment:
Should we rename this to RunOutlineServer? Is there anything tied to Shadowsocks?

Reply:
The `runConfig()` still assumes the config is shadowsocks, but I agree we can at least rename `SSServer` a bit now that it's more localized.